packages/muelu/src/Graph/MatrixTransformation/MueLu_CoalesceDropFactory_def.hpp

```diff
@@ -859,8 +865,10 @@ namespace MueLu {
           ArrayRCP<LO> rows    = ArrayRCP<LO>(numRows+1);
           ArrayRCP<LO> columns = ArrayRCP<LO>(A->getNodeNumEntries());
 
+#ifdef HAVE_MUELU_DEBUG
 	  // DEBUGGING
 	  for(LO i=0; i<(LO)columns.size(); i++) columns[i]=-666;
+#endif
```

Comment:
I'm wondering about the purpose of this. What additional debugging information does it provide to the user?

Reply:
I think it was just meant to initialize to some easily identifiable value.